airgun/views/hostcollection.py

```diff
@@ -230,6 +233,28 @@ def is_displayed(self):
             self.title, exception=False) is not None
 
 
+class HostCollectionManageModuleStreamsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h4[contains(., 'Content Host Module Stream Management')]")
+    table = SatTable(
+        locator='//table',
+        column_widgets={
+            'Name': Text('.//a'),
+            'Actions': ModuleStreamActionDropDown(".//div[contains(@class, 'dropdown')]")
+        },
+    )
+
+    @property
+    def is_displayed(self):
+        """The view is displayed when it's title exists"""
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+    def search(self, query):
```

Comment:
This we can remove completely this function, as it repeat what is in SearchableViewMixin

Reply:
Yes Definitely

Reply:
done!